core/modules/user/user.actions.inc

```diff
@@ -55,8 +55,8 @@ function user_unblock_user_action($account, &$context) {
  * @ingroup actions
  */
 function user_cancel_user_action($account, &$context) {
-  // Save the list of nodes to be deleted in the session. Append to the existing
-  // list if within the last minute, otherwise start a new list of nodes.
+  // Save the list of user accounts to be deleted in the session. Append to the existing
+  // list if within the last minute, otherwise start a new list of user accounts.
```

Comment:
This needs to adhere to the 80-character limit for comments (https://api.backdropcms.org/doc-standards#general):

```suggestion
  // Save the list of user accounts to be deleted in the session. Append to the
  // existing list if within the last minute, otherwise start a new list of user
  // accounts.
```

Reply:
Fixed on next commit.

Reply:
Thanks!